.github/workflows/externalworkloads.yaml

```diff
@@ -12,9 +12,9 @@ concurrency:
   cancel-in-progress: true
 
 env:
+  clusterName: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}-vm
```

Comment:
nit: do we want `-vm` suffix in cluster name?

Reply:
I mostly wanted to have it explicit when we list clusters from GKE if they are part of the default workflow (nothing), multicluster workflow (`mesh`) or external workloads workflow (`vm`). Do you think we should find another suffix?

Reply:
I think that's fine.